external-dns/README.md

```diff
@@ -0,0 +1,7 @@
+# external-dns
+
+### Update procedure
+- Update `Makefile` with the new target commit reference.
+- Check [upstream](https://github.com/kubernetes-sigs/external-dns/tree/master/kustomize) in case the are new resources defined.
```

Comment:
how to check this? would be nice to have a programatic way to do it

Reply:
I did take a look at `git archive` but `github` doesn't support it.
I didn't find anything else sensible, don't wanna parse HTML...

Reply:
Fair